spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -100,20 +148,40 @@ def nav(self, nav_target: str = None) -> Tuple[bool, str]:
         else:
             return False, "No nav target specified, skipping nav"
 
-        print(f"Navigating to {nav_target}")
+        nav_x, nav_y, nav_theta = nav_target_tuple
+        status, message = self.nav(nav_x, nav_y, nav_theta)
+        return status, message
+
+    @multimethod  # type: ignore
+    def nav(self, x: float, y: float, theta=float) -> Tuple[bool, str]:  # noqa
+        """
+        Perform the nav action on the specified navigation target (location)
```

Comment:
Explicit definition of what location means would be good here, i.e. location = x,y,theta

Reply:
I see you do it in args, maybe then just change to "specified navigation target represented as a metric location"

Reply:
Updated